src/validation.cpp

```diff
@@ -1566,6 +1566,22 @@ static bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockInd
         flags |= SCRIPT_VERIFY_NULLDUMMY;
     }
 
+    if (GetBoolArg("-bip148", DEFAULT_BIP148)) {
```

Comment:
That's because the code was *just* refactored to create `gArgs`. New code should probably use it, rather than the wrappers which exist to avoid touching the old code.